modular/stonekeep/code/roguecrafting/carpentry.dm

```diff
@@ -123,6 +123,39 @@
 	time = 4 SECONDS
 	craftdiff = 2
 
+/datum/crafting_recipe/roguetown/carpentry/heater
+	name = "heater shield"
+	result = /obj/item/weapon/shield/heater
+	reqs = list(/obj/item/natural/wood/plank = 2,
+				/obj/item/natural/hide/cured = 1)
+	skillcraft = /datum/skill/craft/carpentry
+	craftdiff = 2
+
+//removes awesome buckler shield crafting
```

Comment:
Commenting out isn't the same as removing. These should be gone entirely, not just commented out.